src/file/functions.rs

```diff
@@ -25,6 +27,17 @@ struct PartitionInfo {
     end: String,
 }
 
+impl Display for Partition {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            Partition::boot => write!(f, "boot"),
+            Partition::rootA => write!(f, "rootA"),
```

Comment:
can you rm rootA here: https://github.com/omnect/omnect-cli/blob/3e4d5f8b221a08c56550ac2fd81433729b383562/src/file/functions.rs#L16

FYI: @mlilien

Reply:
@JanZachmann  imho not a good idea ... we want to read files from rootA
(it doesn't make sense to write to rootA though)